app/views/revision_feedback/index.html.haml

```diff
@@ -1,6 +1,12 @@
 %section
   %h1
     Suggestions
+  %h4
+    Rating
+  %p
+    = @rating
+  %h4
+    More Suggestions
```

Comment:
We can make it admin only for referring to the suggestions on receiving feedback